pkg/system/path_windows_test.go

```diff
@@ -0,0 +1,38 @@
+// +build windows
+
+package system
+
+import "testing"
+
+// TestCheckSystemDriveAndRemoveDriveLetter tests CheckSystemDriveAndRemoveDriveLetter
+func TestCheckSystemDriveAndRemoveDriveLetter(t *testing.T) {
```

Comment:
Could you pls add a test like just `c` and `cpath`? I know that that should be okay, but I feel uneasy about that cases, and other code readers might be too.